packages/theme-chalk/src/table.scss

```
@@ -545,6 +546,12 @@
     }
   }
 
+  @include e(body-header) {
+    position: sticky;
+    top: 0;
+    z-index: 2;
```

我觉得 z-index 换成参数会更好点，因为还需要考虑 cell、固定列、header... 的层级关系。

```suggestion
    z-index: calc(getCssVar('table-index') + 2);
```

---

ok，那 border 的伪类的z-index需要同步改成calc吗

---

需要修改相互影响的 z-index，同时还要在 `common/var.scss` 中定义参数

```diff
+ 'index': getCssVar('index-normal'),
```

---

ok了强哥🙈